src/fetchtastic/downloader.py

```
@@ -221,6 +225,7 @@ def check_promoted_prereleases(
     prerelease_dir = os.path.join(download_dir, "firmware", "prerelease")
     if not os.path.exists(prerelease_dir):
         return False
+    real_prerelease_dir = os.path.realpath(prerelease_dir)
```

Ruff (0.13.1)

Local variable `real_prerelease_dir` is assigned to but never used

Remove assignment to unused variable `real_prerelease_dir`

(F841)